### ISSUE TITLE
04_deeppavlov_chitchat.ipynb -  3 Questions

### ISSUE THREAD

**Issue by tuyoisaza** [deepmipt/DeepPavlov#656]
Hi! Loving de DeepPavlov idea!

In the Chitchat example as suggested, Im tring to run 3 things:

1) Use "teacher_forcing_rate":  It seams that anything other that 0 Activates this. But I see no effect on the loss. It goes just as slow as with 0. I did notice the teacher_forcing_rate = tf.random_uniform(shape=()) and wonder if that is a smarter way to set the value for teacer_forcing_rate, that could be use in the config.

2)  Use "use_attention": True : But it activates a lot of things (kinda hard to follow, sorry...) 
that end up in: InvalidArgumentError (see above for traceback): Restoring from checkpoint failed. This is most likely due to a mismatch between the current graph and the graph from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:
Assign requires shapes of both tensors to match. lhs shape= [550,500] rhs shape= [300,500]
And the suggestions i get for this error is to rewrite some things in Tensorflow... which I would not try and do... Is there a way to fix the example in the Tutorials?

3) At the end of the Tutorial you also suggest: 
To improve the model you can try to use multilayer (use MultiRNNCell) encoder and decoder, try to use attention with trainable parameters (not dot product scoring function).
I wonder how to do this? 
in the encoder def i find 
  encoder_cell = tf.nn.rnn_cell.GRUCell(
                            num_units=cell_size,
                            kernel_initializer=tf.contrib.layers.xavier_initializer(),
                            name='encoder_cell')
But I dont know if that is the correct place, or how to use MultiRNNCell in the Docs at deeppavlov.

I would love to get some help in this tutorial! Thanks!

**Comment by yurakuratov** [comment]
Hi!

This tutorial is a bit outdated, we are going to fix our tutorials section.

I would suggest to check TensorFlow 2.0 Text and Sequences tutorials with more recent TF api: https://www.tensorflow.org/alpha/tutorials/sequences/text_generation
